File: Linguagem-Java/aprendizado/vetores.java
import java.util.Locale;
import java.util.Scanner;

public class vetores {
  public static void main(String[] args) {
  Locale.setDefault(Locale.US);
    Scanner sc = new Scanner(System.in);

    System.out.print("Quantos numeros voce vai digitar? ");
    int N = sc.nextInt();

    double[] vet = new double[N];

    for (int i = 0; i < N; i++) {
      System.out.print("Digite um numero: ");
      vet[i] = sc.nextDouble();
    }
    System.out.println();
    System.out.println("Numeros digitados:");
    for (int i = 0; i < N; i++) {
      System.out.println(String.format("%.1f", vet[i]));
    }

    sc.close();
  }
}

File: Linguagem-C#/Aprendizado/vetores.cs
using System;
using System.Globalization;

namespace vetores
{
  class Program
  {
    static void Main(String[] args) 
    {
      CultureInfo CI = CultureInfo.InvariantCulture;

      int N;
      Console.Write("Quantos numeros voce vai digitar? ");
      N = int.Parse(Console.ReadLine());

      double [] vet = new double[N];
      for (int i = 0; i < N; i++) {
        Console.Write("Digite um numero: ");
        vet[i] = double.Parse(Console.ReadLine(), CI);        
      }
      Console.WriteLine();
      Console.WriteLine("Numeros digitados:");
      for (int i = 0;i < N; i++) {
        Console.WriteLine(vet[i].ToString("F1", CI));
    }
    }
  }
}
